Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -213,7 +213,8 @@ static std::vector<u8> ParseHash(const char* str)
   return hash;
 }
 
-std::vector<RedumpVerifier::PotentialMatch> RedumpVerifier::ScanDatfile(const std::vector<u8>& data)
+std::vector<RedumpVerifier::PotentialMatch> RedumpVerifier::ScanDatfile(const std::vector<u8>& data,
+                                                                        const std::string& system)
```

Comment:
I thought it's all about `std::string_view` now?

Reply:
Using `std::string_view` in this specific case doesn't have any advantages (because the calling function doesn't do any substringing or similar) and actually has a disadvantage in the case where we need to call `c_str()` on it.

Reply:
Just curious but what about making this a pass by value, then using `std::move` ?

Reply:
I don't see why that would be faster unless we need to create a copy of the string for some reason (since that would let us skip making the copy).

Reply:
If it is faster, it's a micro optimization.  No big deal, I just wanted to bring it up since @CookiePLMonster brought up string-view and it has been on my mind